docs/data-sources/compute_instance_list.md

```diff
@@ -16,6 +16,21 @@ Corresponding resource: [exoscale_compute_instance](../resources/compute_instanc
 ```hcl
 data "exoscale_compute_instance_list" "my_compute_instance_list" {
   zone = "ch-gva-2"
+
+  match {
+    attribute = "type"
+    match     = "standard.micro"
+  }
```

Comment:
```suggestion
  match {
    attribute = "type"
    match     = "standard.micro"
  }
```
Why this `match` structure (map) ? Looking at other data sources, it seems to me the usual semantic would be `type = "standard.micro"` ; please correct me if I'm wrong (?)

Checked:
- https://registry.terraform.io/providers/exoscale/exoscale/latest/docs/data-sources/template#arguments-reference
- https://registry.terraform.io/providers/hashicorp/consul/latest/docs/data-sources/acl_policy#argument-reference
- https://registry.terraform.io/providers/hashicorp/vault/latest/docs/data-sources/kv_secret_v2#argument-reference

Reply:
changed it, I'm much happier with it now